lib/mono.h

```diff
@@ -66,12 +66,12 @@
 #include "debug.h"
 void nw_InitTCPLogging(char *ip, unsigned short port);
 void nw_TCPPrintf(int n, char *format, ...);
-#if (!defined(RELEASE)) && defined(MONO)
+#if (!defined(RELEASE)) && (defined(MONO) || defined(LOGGER))
```

Comment:
I don't think we need to keep both, `LOGGER` is enough

Reply:
I wanted to keep both so we could keep the MONO code for historical reasons if we add a logging library. But it's easy enough to resurrect MONO later.